en/ability.json

```diff
@@ -1185,11 +1185,11 @@
   },
   "earthEater": {
     "name": "Earth Eater",
-    "description": "If hit by a Ground-type move, the Pokémon has its HP restored instead of taking damage."
+    "description": "If hit by a Ground-type move, the Pokémon has 1/4 of its maximum HP restored instead of taking damage."
```

Comment:
```suggestion
    "description": "If hit by a Ground-type move, the Pokémon has its HP restored by up to 1/4 of its max HP instead of taking damage."
```